v2/lib/theme/BlogPage/index.js

```diff
@@ -5,46 +5,39 @@
  * LICENSE file in the root directory of this source tree.
  */
 
-/* eslint-disable */
-import React from 'react';
+import React, {useContext} from 'react';
 import {Link} from 'react-router-dom';
 import Helmet from 'react-helmet';
-import Loadable from 'react-loadable';
 import Layout from '@theme/Layout'; // eslint-disable-line
 import BlogPost from '@theme/BlogPost'; // eslint-disable-line
-import Loading from '@theme/Loading';
 
-export default class BlogPage extends React.Component {
-  render() {
-    const {
-      metadata,
-      blogMetadatas,
-      language,
-      children,
-      siteConfig = {},
-    } = this.props;
-    const {posts} = metadata;
+import DocusaurusContext from '../../core/docusaurus-context';
```

Comment:
I think this can be not scalable in the long run (using relative path) like in v1. What if later on we moved tbe core folder slightly

I am thinking of aliasing `@docusaurus` as a component folder that user can import.

So anytime they can do `import DocusaurusContext from '@docusaurus/context`

Or `import GridBlock from '@docusaurus/GridBlock`.

Webpack alias is awesome

Reply:
We should export these on the main library package. Namespace is optional. Yes let's use webpack alias.